scripts/install_catalog.sh

```diff
@@ -85,7 +85,7 @@ run() {
     "${CATALOG_IMAGE}":"${IMAGE_TAG}"
 }
 
-# TODO Create flag for allow remove current service if it exists
+# TODO Create flag for allow not remove current service if it exists
```

Comment:
create a flag to make the service removal optional, in case if already exists

Reply:
done.